dds/DCPS/FilterEvaluator.cpp

```diff
@@ -124,12 +124,26 @@ FilterEvaluator::SerializedForEval::lookup(const char* field) const
   if (iter != cache_.end()) {
     return iter->second;
   }
-  Message_Block_Ptr mb (serialized_->duplicate());
-  // TODO(iguessthislldo): Use Encap
-  Serializer ser(mb.get(),
-    cdr_ ? Encoding::KIND_XCDR1 : Encoding::KIND_UNALIGNED_CDR, swap_);
+  Message_Block_Ptr mb(serialized_->duplicate());
+  Serializer ser(mb.get(), cdr_ ? Encoding::KIND_XCDR1 : Encoding::KIND_UNALIGNED_CDR, swap_);
```

Comment:
```suggestion
  Serializer ser(mb.get(), Encoding::KIND_UNALIGNED_CDR, swap_);
```

This didn't need to change back because the encapsulation header is encoding agnostic and encoding kind will always be overwritten by the one in the encapsulation header if there is one.